addons/skin.estuary/xml/DialogSeekBar.xml

```diff
@@ -22,7 +22,7 @@
 				<visible>!Window.IsVisible(videoosd) + !Window.IsVisible(musicosd)</visible>
 				<animation effect="fade" time="200">VisibleChange</animation>
 				<control type="image">
-					<left>200</left>
+					<left>20</left>
```

Comment:
Left offset here appeared to be wrong, corrected in this PR as discussed on slack since I'm editing in here.

Reply:
Thanks.